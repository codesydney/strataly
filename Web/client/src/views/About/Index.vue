<script>
import './style.scss'
export default {
  name: 'About'
}
</script>

<template>
  <v-container>
    <v-row class="text-center">
      <v-col cols="12">
        <p>About Us</p>
      </v-col>
    </v-row>
  </v-container>
</template>
