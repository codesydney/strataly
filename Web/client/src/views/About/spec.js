import { mount, createLocalVue } from '@vue/test-utils'
import Vuetify from 'vuetify'
import Vue from 'vue'
Vue.use(Vuetify)
import About from './Index.vue'
const localVue = createLocalVue()

describe('CustomCard.vue', () => {
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should have a custom title and match snapshot', () => {
    const wrapper = mount(About, {
      localVue,
      vuetify,
      mocks: {
        $vuetify: { breakpoint: {} }
      }
    })

    // With jest we can create snapshot files of the HTML output
    expect(wrapper.html()).toMatchSnapshot()
  })
})
