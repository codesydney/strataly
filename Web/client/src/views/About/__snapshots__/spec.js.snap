// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomCard.vue should have a custom title and match snapshot 1`] = `
<div class="container">
  <div class="row text-center">
    <div class="col col-12">
      <p>About Us</p>
    </div>
  </div>
</div>
`;
